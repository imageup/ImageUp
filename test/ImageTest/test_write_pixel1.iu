function main() -> int
{
        image: a = read("./images/google-chrome.jpg");

		write_pixel(a, (0,1), (7,8,9));
		write_pixel(a, (1,2), (7,8,9));
		write_pixel(a, (2,3), (7,8,9));
		write_pixel(a, (3,4), (7,8,9));
		write_pixel(a, (4,5), (7,8,9));
		write_pixel(a, (5,6), (7,8,9));
		write_pixel(a, (6,7), (7,8,9));
		save("test.out.jpg", a);
		tuple: t1 = get_pixel(a,(0,1));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(1,2));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(2,3));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(3,4));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(4,5));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(5,6));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
		tuple: t1 = get_pixel(a,(6,7));
		prints("Pixel Written: " + StrParsef(t1[0]) + " " + StrParsef(t1[1]) + " " + StrParsef(t1[2]));
        return 0;
}
