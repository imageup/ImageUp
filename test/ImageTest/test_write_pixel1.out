Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
Pixel Written: 9.00000 8.00000 7.00000
